<?xml version="1.0"?>
<launch>
  <node pkg="joy" exec="joy_node" name="joy_node" />
  <node pkg="jetbot_pro_ros2" exec="teleop_joy" name="teleop_joy_node" output="screen">
	  <param name="x_speed" value="0.3"/>
	  <param name="y_speed" value="0"/>
	  <param name="w_speed" value="1"/>
   </node>
</launch>
